lib/matplotlib/dates.py

```diff
@@ -1304,6 +1304,7 @@ def get_unit_generic(freq):
     def _get_interval(self):
         return self.rule._rrule._interval
 
+    @cbook.deprecated("3.1")
```

Comment:
Please update all deprecations to 3.2. You'll get an approval as a reward. :wink:

Reply:
done